aspnet/tutorials/mvc-with-entity-framework.rst

```diff
@@ -0,0 +1,394 @@
+In **Solution Explorer** find and open *project.json*. In the ``dependencies`` section, begin by typing the following line at the end of the section. 
+
+.. image:: mvc-with-entity-framework/_static/15d2-di-intellisense.png
+	
+You will see that IntelliSense provides coding help as you type. Also, when you save *project.json*, Visual Studio will automaticaly resolve the new package reference.
+
+.. image:: mvc-with-entity-framework/_static/15d3-package-restore.png
+	
+After the above line has been added, the ``dependencies`` section of your *project.json* file will appear as follows:
+
+.. literalinclude:: mvc-with-entity-framework/sample/src/ContosoBooks/project.json
+    :language: aspx-cs
+    :lines: 8-29
+    :emphasize-lines: 21
+
+In **Solution Explorer**, open the *Startup.cs* file. Add the following line of code at the end of the *Configure* method:
+
+.. literalinclude:: mvc-with-entity-framework/sample/src/ContosoBooks/Startup.cs
+    :language: c#
+    :lines: 101
+    :dedent: 12
+
+After the above line has been added, the completed *Startup.cs* file will appear as follows:
+
+.. literalinclude:: mvc-with-entity-framework/sample/src/ContosoBooks/Startup.cs
+    :language: c#
+    :emphasize-lines: 101
+
+Notice in *ConfigureServices* the app calls ``Configuration.Get`` to get the database connection string. During development, this setting comes from the *config.json* file. When you deploy the app to a production environment, you set the connection string in an environment variable on the host. If the Configuration API finds an environment variable with the same key, it returns the environment variable instead of the value that is in *config.json*.
+	
+Build the web application
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To make sure that all the classes and changes to your Web application work, you should build the application.
+
+From the **Build** menu, select **Build Solution**.
+
+The **Output** window is displayed, and if all went well, you see a succeeded message.
+
+.. image:: mvc-with-entity-framework/_static/19-vs-build-output.png
+
+If you run into an error, re-check the above steps. The information in the **Output** window will indicate which file has a problem and where in the file a change is required. This information will enable you to determine what part of the above steps need to be reviewed and fixed in your project.
+
+Using data migrations to create the database
+--------------------------------------------
+Data migrations in EF are used to perform model updates throughout your entire application. By initially using data migrations to create your database, you can modify your database after the model has changed with simple steps. This will allow you to build and maintain your web app more efficiently. The alternative to data migrations, where model or schema changes are required after the database has been created, involves recreating your entire database.
```

Comment:
That's not the alternative. migrations allows schema/model changes after the db has been created. You could write
Alternatively, early in development you might want to simply drop and recreate the DB rather than do migrations.